### PR TITLE
ROC-7040: Allow caseworkers to resend new PINs via email

### PR DESCRIPTION
**Pull request by KiranVarma02**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-7040

### Change description ###

Allow caseworkers to resend new PINs via email

### CODE REVIEW

File: definition/data/sheets/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -245,5 +245,6 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ResumeClaimPaymentCitizen", "UserRole": "citizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ResumeClaimPaymentCitizen", "UserRole": "caseworker-cmc", "CRUD": "C"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateCitizenClaim", "UserRole": "citizen", "CRUD": "CR"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateCitizenClaim", "UserRole": "caseworker-cmc", "CRUD": "CR"}
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateCitizenClaim", "UserRole": "caseworker-cmc", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ResendPin", "UserRole": "caseworker-cmc", "CRUD": "CR"}
```
Comment (andrewwa-kainos) [inline definition/data/sheets/AuthorisationCaseEvent.json]: I suggest `ResetPin` instead, as `Resend` sounds like sending the same one again

Comment (KiranVarma02) [reply definition/data/sheets/AuthorisationCaseEvent.json]: i agree, i thought that too but just followed what it says in the ticket. I guess we should put whatever makes more sense for case worker, i am guessing who ever created this ticket would have thought about that. What do others think?

Comment (daveoh) [reply definition/data/sheets/AuthorisationCaseEvent.json]: I agree with @andrewwa-kainos. Seeing as the event Id is not shown to the end user I would think that we should be ok to decide on the name of this within the team. Happy to be overruled though!

Comment (KiranVarma02) [reply definition/data/sheets/AuthorisationCaseEvent.json]: agree, done.

**Pull request merged**